independent-publisher-connectors/Nifty Gateway/readme.md

```diff
@@ -0,0 +1,20 @@
+# Nifty Gateway
+Nifty Gateway if a platform for buying and selling NFTs.  This connector allows a user to pull this data into their app, flow, or report.
+
+## Publisher: Roy Paar
+
+## Prerequisites
+None.
+
```

Comment:
There is no Obtaining Credentials section that is required. Please refer to this [link](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/Independent%20Publisher/readme.md).

Reply:
Fixed